driverkit/config/17f5df52a7d9ed6bb12d3b1768460def8439936d/ubuntu-aws_5.4.0-1057-aws_62.yaml

```
@@ -0,0 +1,6 @@
+kernelversion: 62
+kernelrelease: 5.4.0-1059-aws
+target: ubuntu-aws
+output:
+  module: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_ubuntu-aws_5.4.0-1059-aws_62.ko
+  probe: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_ubuntu-aws_5.4.0-1059-aws_62.o
```

```suggestion
kernelrelease: 5.4.0-1057-aws
target: ubuntu-aws
output:
  module: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_ubuntu-aws_5.4.0-1057-aws_62.ko
  probe: output/17f5df52a7d9ed6bb12d3b1768460def8439936d/falco_ubuntu-aws_5.4.0-1057-aws_62.o
```

Hey @yogisec 

the config filename does not match its content. I guess you wanted to configure 5.4.0-105**7**-aws here (and not 5.4.0-105**9**-aws, right?

---

Whoops sorry about that. 1059 is correct...I just messed up and did not change the file name completely after copying it. The file names have been updated now.